CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/reporting/PluginReport.java

```diff
@@ -165,17 +170,22 @@ public String getPluginDescription() {
      *
      * @return the current message from this plugin.
      */
-    public String getMessage() {
-        return message;
+    public String getLastMessage() {
+        if ((executionStage.equals(PluginExecutionStageConstants.COMPLETE) || executionStage.equals(PluginExecutionStageConstants.STOPPED)) && !this.messageLog.isEmpty()){
+            return this.messageLog.isEmpty() ? "" : this.messageLog.get(this.messageLog.size()-1);
```

Comment:
the boolean check here isn't required as the if condition preceding it means it can only enter here if `!this.messageLog.isEmpty()` is true (in other words, the boolean check here will always return false)